backend/app/routes/admin/getAdmins.js

```diff
@@ -0,0 +1,53 @@
+const to = require('await-to-js').default;
+const Admin = require('../../models/Admin');
+const { ErrorHandler } = require('../../../helpers/error');
+const constants = require('../../../constants');
+
+const createPipeline = (adminType, page) => {
+  const pipeline = [
+    {
+      $project: {
+        _id: 0,
+        firstName: '$firstName',

```

Comment:
just a quick heads up, since the fields are matching you can alternatively do like this,
`firstName : 1,`
Please change these to this format as it is a little more effective in prod
For creating pipelines you can try it out first on mongoDB Compass, they have good visualization.